src/program.js

```diff
@@ -184,6 +184,10 @@ Example: $0 --help run.
         demand: false,
         type: 'string',
       },
+      'no-reload': {
```

Comment:
the option would need to have a different name because you can't set a flag to false on the command line. I think it's ok if it's slightly broken since the tool is still experimental. I won't cut a new release of web-ext until reloading works better.